pkg/cli/image/info/info.go

```diff
@@ -99,6 +99,9 @@ func (o *InfoOptions) Complete(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("info expects at least one argument, an image pull spec")
 	}
 	o.Images = args
+	if err := o.FilterOptions.BookendRegex(); err != nil {
```

Comment:
I would expect this to be handled inside Complete() by a boolean on FilterOptions set in NewInfo/NewAppend (`filter options ExpectSingle`).

Reply:
ok, will do that instead

Reply:
done